library/tasks/SequenceDataWithMoleculeTagMetrics.wdl

```diff
@@ -0,0 +1,88 @@
+
+task CalculateGeneMetrics {
```

Comment:
Is there a reason these tasks are grouped into one wdl instead of separate ones? I think most of the wdl files in the `tasks` directory are single tasks.

Reply:
I have been grouping related tasks in task WDLs for 3' pipelines. See Kallisto and Star for examples. If we dislike this we can separate.

Reply:
Oh okay! I think it's fine to keep related tasks together.